packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -89,6 +92,20 @@ export class BlockList extends Component {
 		return renderAppender && blockClientIds.length > 0;
 	}
 
+	getVerticalAlignmentRemap( alingment ) {
+		if ( ! alingment ) return;
+
+		const mapping = {
+			top: 'flex-start',
+			center: 'center',
+			bottom: 'flex-end',
+		};
+
+		const newAlingment = mapping[ alingment ];
+		if ( ! newAlingment ) return;
+		return { justifyContent: alingment };
```

Comment:
Ok, it's because instead `return { justifyContent: alingment }` it should return `return { justifyContent: newAlingment }`. Missmatch after refactor. Fixed!